### DISCUSSION TITLE
How to serve a iles generated site over Node request?

### DISCUSSION THREAD

**Discussion by paraself** [ElMassimo/iles#182] [Q&A] [answered]
Hello, I'd like to do SSR on our Node server, similar to this:

https://github.com/frandiox/vite-ssr/blob/master/examples/node-server/index.js

1. The request reaches the api
2. Fetch global data according to the target route
3. render the site to html ( different route produces different html )
4. response the html to client

Sorry I did not find the guide on how to do these steps on iles documentation. Could you please share how to do these in iles? Many thanks

**Comment by ElMassimo** [comment, marked as answer]
Hi Yves!

îles does not support SSR at the moment, ideally the site is built before deployment ([Jamstack](https://jamstack.org/generators/)).

I'd suggest you try [Nuxt](https://v3.nuxtjs.org/) if you need to serve different content for different users.

_If what you want is to serve the build-time generated HTML through a Node server, most servers support some kind of [static files](https://expressjs.com/es/starter/static-files.html) middleware._
